src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -1618,9 +1619,30 @@ public static LivingEntity getLivingEntity(ModElement modElement, Random random,
 		livingEntity.immuneToWither = !_true;
 		livingEntity.hasSpawnEgg = !_true;
 		livingEntity.xpAmount = 8;
-		livingEntity.ridable = _true;
+		livingEntity.ridable = !_true;
 		livingEntity.canControlStrafe = !_true;
 		livingEntity.canControlForward = _true;
+		livingEntity.canTrade = _true;

```

Comment:
Test fail because if you set canTrade to true, LivingEntityGUI#updateTradingConditions overrides values of fields such as aiBase, ridable, etc. to fixed value, causing UI tests to fail as fields change / GE definition changes when opening and saving ME GUI editor.

To fix this, you need to add `if (livingEntity.ridable) { ...` below this line and rewrite needed parameters to values accepted / set to in updateTradingConditions

However, make sure other combinations are still properly tested. In other words, if _true makes canTrade but also enabled ridable which is later disabled, ridable entities will never be tested in this case so make sure to cover this.

![image](https://github.com/MCreator/MCreator/assets/16374228/7ec3efc4-7414-43aa-93f9-e7ac76625280)

Reply:
What do I put inside the `if (livingEntity.ridable) { ...`?

Reply:
> rewrite needed parameters to values accepted / set to in updateTradingConditions

Reply:
Why did you mark this resolved while this:

1. is not resolved
2. reviewers resolve comments

?

Reply:
`if (livingEntity.ridable && livingEntity.canTrade) {`

is not same as what I asked:

`if (livingEntity.ridable)`

and tests are not happy

Reply:
Now you put

`if (livingEntity.ridable) {`

which is not same as what I asked

`if (livingEntity.ridable)`

and tests are not happy

Reply:
> Why did you mark this resolved while this:
> 
> 1. is not resolved
> 2. reviewers resolve comments
> 
> ?

Excuse me I assumed because of the...
<img width="82" alt="Capture d’écran 2023-09-15 à 10 44 51" src="https://github.com/MCreator/MCreator/assets/49318596/c6a829f0-3701-44d9-bdf3-2c2d53bc6767">

Reply:
Outdated just indicates line shown in the GitHub review comment is no longer up-to-date

![image](https://github.com/MCreator/MCreator/assets/16374228/3c72dc5d-b3fd-41cc-a8fa-6ca0f197abd0)

Reply:
Ok, excuse me, I meant

`if (livingEntity.canTrade) {`

I am sorry for the confusion I have caused here

Reply:
Ok done

Reply:
Fingers crossed tests are happy now

Reply:
Nope